lib/rubocop/cop/rails/tag_instead_of_content_tag.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks that `tag` is used instead of `content_tag`
+      # because `content_tag` is legacy syntax.
+      #
+      # @example
+      #  # bad
+      #  content_tag(:p, 'Hello world!')
+      #  content_tag(:br)
+      #
+      #  # good
+      #  tag.p('Hello world!')
+      #  tag.br
+      class TagInsteadOfContentTag < Cop
+        include RangeHelp
+        extend TargetRailsVersion
+
+        minimum_target_rails_version 5.1
+
+        BAD_METHOD = 'content_tag'
+        GOOD_METHOD = 'tag'
+        MSG = 'Use `tag` instead of `content_tag`.'
+
+        def on_send(node)
+          method_name = node.method_name.to_s
+
+          return if method_name != BAD_METHOD
```

Comment:
Can you use `method?`?

```suggestion
          return unless node.method?(:content_tag)
```